### PR TITLE
Added basic usecase

### PR DESCRIPTION
**Pull request by abhi-singhs**

### CODE REVIEW

File: src/main/java/com/browserstack/Test.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        capabilities.setCapability("app", "WikipediaApp");
+        capabilities.setCapability("name", "BrowserMob with BrowserStack App-Automate");
+        capabilities.setCapability("browserstack.debug", "true");
+        capabilities.setCapability("browserstack.networkLogs", "true");
+        capabilities.setCapability("browserstack.acceptInsecureCerts", "true");
+        capabilities.setCapability("browserstack.local", "true");
+        capabilities.setCapability("browserstack.localIdentifier", "Test1");
+
+        AndroidDriver<AndroidElement> driver = new AndroidDriver<>(
+                new URL("http://" + username + ":" + accessKey + "@hub.browserstack.com/wd/hub"), capabilities);
```
Comment (samirans89) [inline src/main/java/com/browserstack/Test.java]: - use https
- keep username and accessKey in caps.

File: src/main/java/com/browserstack/Test.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+        Thread.sleep(10000);
+
+        System.out.println("Getting HAR file");
+
+        List<HarEntry> entries = proxy.getHar().getLog().getEntries();
+        for (HarEntry harEntry : entries) {
+            System.out.println(harEntry.getRequest().getUrl());
+        }
+        FileOutputStream fos = new FileOutputStream("test.har");
```
Comment (samirans89) [inline src/main/java/com/browserstack/Test.java]: Define filename as the start of the file as a Const

**Review by samirans89 — CHANGES_REQUESTED** [review]

File: src/main/java/com/browserstack/Test.java (hunk 1 above)
Comment (abhi-singhs) [reply src/main/java/com/browserstack/Test.java]: Done

File: src/main/java/com/browserstack/Test.java (hunk 2 above)
Comment (abhi-singhs) [reply src/main/java/com/browserstack/Test.java]: Done

**Review by samirans89 — APPROVED** [review]

**Pull request merged**